azure-pipelines/psf-public-nuget-build.yml

```diff
@@ -0,0 +1,352 @@
+                    include\psf_framework.h
+                    include\psf_runtime.h
+                    include\psf_utils.h
+                    include\utilities.h
+                    include\win32_error.h
+                  TargetFolder: $(Build.BinariesDirectory)
+                  OverWrite: true
+              - task: CopyFiles@2
+                displayName: Stage Libraries
+                condition: and(succeeded(), eq(variables['BuildConfiguration'], 'release'))
+                inputs:
+                  SourceFolder: $(system.defaultworkingdirectory)
+                  Contents: "*\\Release\\PSFRuntime*.lib"
+                  TargetFolder: $(Build.BinariesDirectory)
+                  OverWrite: true
+              - task: CopyFiles@2
+                displayName: Stage Binaries
+                condition: and(succeeded(), eq(variables['BuildConfiguration'], 'release'))
+                inputs:
+                  SourceFolder: $(system.defaultworkingdirectory)
+                  Contents: |-
+                    *\Release\PSFLauncher*.exe
+                    *\Release\PSFRunDll*.exe
+                    *\Release\PSFRuntime*.dll
+                    *\Release\FileRedirectionFixup*.dll
+                    *\Release\RegLegacyFixups*.dll
+                    *\Release\EnvVarFixup*.dll
+                    *\Release\TraceFixup*.dll
+                    *\Release\WaitForDebuggerFixup*.dll
+                    *\Release\TraceLoggerLib.dll
+                    *\Release\StartingScriptWrapper.ps1
+                    *\Release\DynamicLibraryFixup*.dll
+                  TargetFolder: $(Build.BinariesDirectory)
+                  OverWrite: true
+              - task: CopyFiles@2
+                displayName: StageXmlToJsonFiles
+                condition: and(succeeded(), eq(variables['BuildConfiguration'], 'release'))
+                inputs:
+                  SourceFolder: xmlToJsonConverter
+                  Contents: "*"
+                  TargetFolder: $(Build.BinariesDirectory)
+                  OverWrite: true
+              - task: CopyFiles@2
+                displayName: StagePSFMonitor
+                condition: and(succeeded(), eq(variables['BuildConfiguration'], 'release'))
+                inputs:
+                  SourceFolder: $(system.defaultworkingdirectory)
+                  Contents: |-
+                    *\Release\amd64\*
+                    *\Release\x86\*
+                    *\Release\PsfMonitor*.exe
+                    *\Release\Dia2Lib.dll
+                    *\Release\Microsoft.Diagnostics.FastSerialization.dll
+                    *\Release\Microsoft.Diagnostics.Tracing.TraceEvent.dll
+                    *\Release\OSExtensions.dll
+                    *\Release\TraceReloggerLib.dll
+                  TargetFolder: $(Build.BinariesDirectory)
+                  OverWrite: true
+
+              # sign binaries
+              - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
```

Comment:
ESRP codesigning task version looks to be different for sign binaries and sign nuget. Let's mantain one version for both